### PR TITLE
Continue best practices for libpacemaker

### PR DESCRIPTION
**Pull request by kgaillot**
This continues the project to bring libpacemaker up to current development guidelines, focusing on functions before "stage 5" in pcmk_sched_allocate.c

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , here's another low-priority review. Only one more after this for a while ...

File: lib/pacemaker/pcmk_sched_allocate.c [hunk 1]
```diff
@@ -215,6 +215,25 @@ apply_stickiness(pe_resource_t *rsc, pe_working_set_t *data_set)
...
+/*!
+ * \internal
+ * \brief Apply shutdown locks for all resources as appropriate
+ *
+ * \param[in] data_set  Cluster working set
+ */
+static void
+apply_shutdown_locks(pe_working_set_t *data_set)
+{
+    if (pcmk_is_set(data_set->flags, pe_flag_shutdown_lock)) {
```
Comment (clumens) [inline lib/pacemaker/pcmk_sched_allocate.c:227]: If you switch this to `if (!pcmk_is_set(...))` and return, then you can un-indent everything after.

File: lib/pacemaker/pcmk_sched_utilization.c [hunk 2]
```diff
@@ -441,3 +441,23 @@ pcmk__create_utilization_constraints(pe_resource_t *rsc, GList *allowed_nodes)
...
+ * \internal
+ * \brief Output node capacities if enabled
+ *
+ * \param[in] desc      Prefix for output
+ * \param[in] data_set  Cluster working set
+ */
+void
+pcmk__show_node_capacities(const char *desc, pe_working_set_t *data_set)
+{
+    if (pcmk_is_set(data_set->flags, pe_flag_show_utilization)) {
```
Comment (clumens) [inline lib/pacemaker/pcmk_sched_utilization.c:455]: Likewise with inverting the test and un-indenting here.

**Comment by kgaillot** [comment]
retest this please

File: lib/pacemaker/pcmk_sched_allocate.c [hunk 3]
```diff
@@ -330,31 +330,44 @@ allocate_resources(pe_working_set_t *data_set)
...
+/*!
+ * \internal
+ * \brief Schedule fail count clearing on online nodes if resource is orphaned
+ *
+ * \param[in] rsc       Resource to check
+ * \param[in] data_set  Cluster working set
+ */
 static void
-cleanup_orphans(pe_resource_t * rsc, pe_working_set_t * data_set)
+clear_failcounts_if_orphaned(pe_resource_t *rsc, pe_working_set_t *data_set)
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply lib/pacemaker/pcmk_sched_allocate.c:349]: Yes but it makes the function usable with g_list_foreach() :)

File: lib/pacemaker/pcmk_sched_allocate.c (hunk 1 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_sched_allocate.c]: added commit on top for this and the other similar one

**Comment by clumens** [comment]
Looks good to me.

**Pull request merged**